projects/services-layers/src/lib/types/Layers.ts

```diff
@@ -124,6 +132,7 @@ export interface IRasterLayerOptions extends ILayerOptions {
   /** check if the service supports this tilesize */
   tileSize?: number;
   type: TRasterLayertype;
+  crossOrigin?: CrossOriginType;
```

Comment:
If this attribute is able for both (IRasterLayerOptions/IVectorLayerOptions) please remove it from them and add it to the general ILayerOptions because both are extended from it.